### ISSUE TITLE
 ErrorException: Expected array for frame 0

### ISSUE THREAD

**Issue by andrey-helldar** [Seldaek/monolog#1199]
When the code throws out the event, in the console I see an error processing it:

```
ErrorException  : Expected array for frame 0

  at /home/vagrant/code/example.com/vendor/monolog/monolog/src/Monolog/Formatter/LineFormatter.php:144
    140|         }
    141| 
    142|         $str = '[object] ('.get_class($e).'(code: '.$e->getCode().'): '.$e->getMessage().' at '.$e->getFile().':'.$e->getLine().$previousText.')';
    143|         if ($this->includeStacktraces) {
  > 144|             $str .= "\n[stacktrace]\n".$e->getTraceAsString()."\n";
    145|         }
    146| 
    147|         return $str;
    148|     }

  Exception trace:

  1   Exception::getTraceAsString()
      /home/vagrant/code/example.com/vendor/monolog/monolog/src/Monolog/Formatter/LineFormatter.php:144

  2   Monolog\Formatter\LineFormatter::normalizeException(Object(ErrorException))
      /home/vagrant/code/example.com/vendor/monolog/monolog/src/Monolog/Formatter/NormalizerFormatter.php:95

  Please use the argument -v to see more details.
```

Test code:
```php
Artisan::command('foo:bar', function () {
    $foo = $bar;
});
```

Error handling of exceptions in the console occurs whenever there is no catching of exceptions.

I use Laravel Framework 5.7 with `monolog/monolog` package `1.23.0` version.
PHP: `7.2.9`
OS: `ubuntu 18.04.1` on Laravel Homestead.

**Comment by andrey-helldar** [comment]
Probably the problem was not in the package, but in the [Laravel framework](https://github.com/laravel/framework/releases/tag/v5.7.8). There is no such error in release [5.7.8](https://github.com/laravel/framework/releases/tag/v5.7.8).

**Comment by andrey-helldar** [comment]
The problem persists even in the most recent version of the framework - `5.7.12
![2018-11-05_20-46-09](https://user-images.githubusercontent.com/10347617/48016171-f3899400-e13b-11e8-8d61-bba99a79442c.png)
If you comment out the 144 line, then the monolog returns a real error, not its own.

**Comment by andrey-helldar** [comment]
This is both funny and not. I found a problem.
My project uses a [package](https://github.com/andrey-helldar/notify-exceptions/blob/master/src/Models/ErrorNotification.php#L36) that is responsible for saving the exceptions to the database, then creating an application in jira and notifying in slack. This error was generated by the method of serialization of exceptions for writing. This is necessary in order to work with it as with the class of the executor during deserialization.
Why did the error occur in freshly installed applications? I first installed the entire package set ...
